### ISSUE TITLE
Ethnicity field choices

### ISSUE THREAD

**Issue by cxrodgers** [safwanrahman/jobtalk_beta#6]
Here are the choices from the mockup.

Asian
Black
Caucasian
Hispanic/Latino
Indigenous
Other

Currently these are not coded into the choices. It is easy to add them, but first I would like to make sure these are the choices that the developers want. Is it not the case that, for instance, many people identify as both Hispanic (ethnicity) and White (race)? And there are many other such situations. Frankly I think this is a minefield because any set of options we provide may be problematic for some people, or even the fact that we ask at all.

One option would be a "check all that apply" and then a large cloud of many possible terms (Asian/East Asian/South Asian/Indian/Pacific Islander/.......)

Note that here many developers recommend against soliciting this information at all:
https://ux.stackexchange.com/questions/73489/where-is-the-best-place-to-ask-the-user-their-ethnicity
Though this particular application may be a special case as workplace response to racial identity is one of the main things we want to discuss.

**Comment by aromartins** [comment]
Thanks for pointing this out - yes we would like to include this as it is in fact one of the main issues we hope to bring to the table. I fully agree with the approach of selecting all that apply and have a larger number of options.

**Comment by clancykelly** [comment]
Yeah we do want gender/ethnicity to be able to track which universities are doing best. It is controversial. Maybe the best solution is to have a 'don't want to say' option in case someone is really against giving that info. The ethnicity/gender info won't be linked to the publicly available reviews, they should just be used for internal calculation of how xyz university is doing in keeping xyz population happy. 
But yeah we will do some research and find the best terms to use for this!
In the mockup we have this qualification for the gender/ethnicity questions:

*Why are we asking this?
There is a long-documented disparity in hiring and compensation for individuals in different genders/ethnic groups, which is particularly pronounced in academia. This information will not be made public in any way on this site, but we will anonymize it and use it for quantitative evaluations of academic workplaces.

**Comment by aromartins** [comment]
This may be useful: https://gist.github.com/ag14spirit/255ca96e7c6c2f0c3845e2ade14de245

**Comment by clancykelly** [comment]
That might be a little too detailed for our purposes. The general consensus I'm seeing is, along these lines:
http://166.78.170.144/sites/default/files/Race%20and%20Ethnicity%20Tables%20for%20MU.pdf
https://ohr.gatech.edu/resources/employeerelations/race

We have to be thoughtful about race vs ethnicity. So perhaps the question could read "Please provide us some information about your race/ethnicity" and the values could be:

Caucasian: Not Hispanic/Latinx
Caucasian: Hispanic/Latinx
Indigenous/Native
Pacific Islander
Asian
Black or African American
Other/Unknown
Would prefer not to answer
